bin-src/tasks/verify.js

```diff
@@ -85,11 +88,16 @@ export const createBuild = async (ctx, task) => {
     transitionTo(runOnlyFiles)(ctx, task);
   }
 
-  const { createBuild: build } = await client.runQuery(TesterCreateBuildMutation, {
+  const { createBuild: build } = await ctx.client.runQuery(TesterCreateBuildMutation, {
     input: {
       ...commitInfo,
+      rebuildForBuildId,
       ...(only && { only }),
       ...(onlyStoryFiles && { onlyStoryFiles: Object.keys(onlyStoryFiles) }),
+      ...(turboSnap && { turboSnapEnabled: !turboSnap.bailReason }),
```

Comment:
Are we calling this `onlyStoryFiles` or `turboSnap` on the backend? Isn't it kind of confusing to use both?

Reply:
Well, I tried to avoid calling it `turboSnap` in the CLI for a while, but in the end that didn't seem right. Adding `bailReason` seems too generic (bail on what?) for an API, so I decided to name it what it is.

Initially, `turboSnapEnabled` would just be an indicator of whether `--only-changed` was set, regardless of bailing (i.e. they attempted to use TS). I then decided this was confusing and `turboSnapEnabled` should just be true when it's actually going through with it (not bailing).

I guess we can technically do without the `turboSnapEnabled` field and just infer it from the presence of `onlyStoryFiles` and omission of `turboSnapBailReason`. WDYT?

Reply:
I get it, thanks @ghengeveld